todo_app/templates/index.html

```diff
@@ -8,6 +8,26 @@ <h1 class="display-4">To-Do App</h1>
   </div>
 
   <form action="add_todo" method="post">
+    <label for="name>"Enter New Todo: </label>
+    <input type="text" name="title" required />
+    <input type="submit" value="Add New Todo" />
+  </form>
+
+  <div class="row">
+    <div class="col-md-12">
+      <h2>items<h2>
+      <ul class="list.group mb-4">
+       {% for item in items %}
+        <li>{{ item.name}} - {{item.status}}</li>
+        <form action="complete -item/{{item.id}}"method="post"
+          <button><Complete</button>
+        <form>
```

Comment:
This should be a closing tag here, without it Chrome etc. might be able to guess but may lead to weird behaviour
```suggestion
        </form>
```